picard/util/__init__.py

```diff
@@ -512,10 +509,6 @@ def load_json(data):
     return json.loads(__convert_to_string(data))
 
 
-def parse_json(reply):
-    return load_json(reply.readAll())
```

Comment:
As discussed on IRC this is a potentially plugin breaking API, but:

- No plugin is currently accessing this (and probably shouldn't but use `load_json` instead)
- It is very specific to the webservice (accepting a response object)
- Plugin can be easily adapted if needed

Reply:
I'll remove those commits.

Reply:
Done, and rebased